scripts/run-alphanet-relay.sh

```diff
@@ -73,13 +73,20 @@ else
   RELAY_BASE_PATH="$RELAY_BASE_PREFIX-relay-$RELAY_INDEX"
 fi
 
+if [ -z "$POLKADOT_VERSION" ]; then
+  POLKADOT_VERSION="sha-`egrep -o 'paritytech/polkadot.*#([^\"]*)' Cargo.lock | \
+    head -1 | sed 's/.*#//' |  cut -c1-8`"
+fi
+
+echo "Using Polkadot revision #${POLKADOT_VERSION}"
+
```

Comment:
So it detects the polkadot version from the cargo.lock file right? Pretty slick. :+1: